core/blockchain_repair_test.go

```diff
@@ -1779,6 +1779,7 @@ func testRepair(t *testing.T, tt *rewindTest, snapshots bool) {
 			SnapshotLimit:  0, // Disable snapshot by default
 		}
 	)
+	defer engine.Close()
```

Comment:
Is it necessary to close the `FullFaker` ethash engine? There is nothing to do in the Close function

Reply:
You're right it can't be necessary, but doing it is more correct, so I'd rather just keep it htere